src/Stratis.Bitcoin.IntegrationTests.Common/Extensions/FullNodeTestBuilderExtension.cs

```diff
@@ -98,5 +99,21 @@ public static IFullNodeBuilder ReplaceTimeProvider(this IFullNodeBuilder fullNod
 
             return fullNodeBuilder;
         }
+
+        public static IFullNodeBuilder ReplaceService<I,T>(this IFullNodeBuilder fullNodeBuilder)
```

Comment:
I think we have a method for this already

Reply:
That implementation takes a class instance as argument which would require the class constructor to be called first. Unfortunately that would require many arguments to be passed which is far less convenient.